<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseStructureWidget</class>
 <widget class="QWidget" name="DatabaseStructureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="btnType">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resurses.qrc">
             <normaloff>:/img/select_type.png</normaloff>:/img/select_type.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutTree"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnSetItem">
       <property name="text">
        <string> &gt; </string>
       </property>
       <property name="icon">
        <iconset resource="resurses.qrc">
         <normaloff>:/img/chevron-right.svg</normaloff>:/img/chevron-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSetItems">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="resurses.qrc">
         <normaloff>:/img/chevron-double-right.svg</normaloff>:/img/chevron-double-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveItem">
       <property name="text">
        <string> &lt; </string>
       </property>
       <property name="icon">
        <iconset resource="resurses.qrc">
         <normaloff>:/img/chevron-left.svg</normaloff>:/img/chevron-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveItems">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="resurses.qrc">
         <normaloff>:/img/chevron-double-left.svg</normaloff>:/img/chevron-double-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resurses.qrc"/>
 </resources>
 <connections/>
</ui>
